compose/cli-command.md

```diff
@@ -0,0 +1,14 @@
+---
+description: Compose in the Docker CLI Tech Preview
+keywords: documentation, docs, docker, compose, containers
+title: Compose in the Docker CLI Tech Preview
+---
+
+The Docker CLI now supports the compose command to include (most of) the docker-compose features and flags in the Docker CLI, without the need for a separate tool.
+
+You can just replace the dash with a space when you use docker-compose to just adopt “docker compose”. You can also use them interchangeably, so that you are not locked-in with the new compose command and can still use docker-compose if needed.
```

Comment:
```suggestion
You can just replace the dash (`-`) with a space when you use `docker-compose` to just adopt `docker compose`. You can also use them interchangeably, so that you are not locked-in with the new `compose` command and can still use `docker-compose` if needed.
```